op-deployer/pkg/deployer/state/intent.go

```diff
@@ -46,9 +46,9 @@ var emptyAddress common.Address
 var emptyHash common.Hash
 
 type SuperchainProofParams struct {
-	WithdrawalDelaySeconds          uint64 `json:"withdrawalDelaySeconds" toml:"withdrawalDelaySeconds"`
-	MinProposalSizeBytes            uint64 `json:"minProposalSizeBytes" toml:"minProposalSizeBytes"`
-	ChallengePeriodSeconds          uint64 `json:"challengePeriodSeconds" toml:"challengePeriodSeconds"`
+	WithdrawalDelaySeconds          uint64 `json:"faultGameWithdrawalDelay" toml:"withdrawalDelaySeconds"`
+	MinProposalSizeBytes            uint64 `json:"preimageOracleMinProposalSize" toml:"minProposalSizeBytes"`
+	ChallengePeriodSeconds          uint64 `json:"preimageOracleChallengePeriod" toml:"challengePeriodSeconds"`
```

Comment:
Why are the json but not the toml field names changed?

Reply:
TBH this struct is never serialized to TOML. I'll update the TOML fields as well for consistency.